sourcegraph/sourcegraph.proto

```diff
@@ -608,6 +612,21 @@ message ReposListTagsOp {
 	RepoListTagsOptions opt = 2;
 }
 
+message ReposListCommittersOp {
+	RepoSpec repo = 1 [(gogoproto.nullable) = false];
+	RepoListCommittersOptions opt = 2;
+}
+
+message RepoListCommittersOptions {
+	string revision = 3;
```

Comment:
If you look through this file, "revision" field comes up 5 times (in `CombinedStatus`, `RepoRevSpec`, `DefsListExamplesOp`, `Example` and `RevToken`) and it is called simply `rev` each time.

I think it's be better to be consistent and also name it `rev` here.